### DISCUSSION TITLE
running solara from command line

### DISCUSSION THREAD

**Discussion by malawleyGH** [widgetti/solara#645] [Q&A] [answered]
Please help with this problem, Thanks in advance

From the quickstart page, this line should start the browser

$ solara run sol.py
Solara server is starting at http://localhost:8765

but when I try the same, I get this error

PS C:\Users\malaw\Documents\MSDS\ship\model\virus_on_network\virus_on_network> solara run app.py                                                  
solara : The term 'solara' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the 
name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ solara run app.py
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (solara:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Note that solara is installed:

Name: solara
Version: 1.32.1
Summary:
Home-page:
Author:
Author-email: "Maarten A. Breddels" <maartenbreddels@gmail.com>
License: The MIT License (MIT)

        Copyright (c) 2022 Maarten A. Breddels

        Permission is hereby granted, free of charge, to any person obtaining a copy
        of this software and associated documentation files (the "Software"), to deal
        in the Software without restriction, including without limitation the rights
        to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
        copies of the Software, and to permit persons to whom the Software is
        furnished to do so, subject to the following conditions:

        The above copyright notice and this permission notice shall be included in
        all copies or substantial portions of the Software.

        THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
        IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
        FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
        AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
        LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
        OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
        THE SOFTWARE.
Location: c:\users\malaw\anaconda3\envs\sir\lib\site-packages
Requires: solara-server, solara-ui
Required-by: Mesa

I am running in a conda environment SIR. There is no executable for solara in the script directory: C:\Users\malaw\anaconda3\envs\SIR\Scripts

**Comment by iisakkirotko** [comment, marked as answer]
Hey @malawleyGH!

Did you upgrade from an older version of solara, or install in a fresh environment from scratch? We recently (see "Version 1.31.0" [here](https://solara.dev/changelog)) split the package to different subpackages. This could result in the `solara` command being removed, and would be fixed by reinstalling `solara-server`.

**Reply by maartenbreddels** [reply]
PS: I made a pinned issue for this https://github.com/widgetti/solara/issues/614

**Reply by malawleyGH** [reply]
Yes, this is exactly what happened,  upgraded and command stopped working. So I unistalled mesa and solara and reinstalled, maybe I did this a couple of times, and eventually it began working. Thanks for your help.
